aptos/sources/Governance.move

```diff
@@ -20,45 +21,32 @@ module Wormhole::Governance {
         //guardians:          Table::Table<u8, Guardian>,
         guardians:          vector<Guardian>,
     }
-    
-    struct Guardian has key, store, drop{
-        key:       vector<u8>,
-    }
 
-    struct GuardianSet has key {
-        index:     u64, 
-        guardians: vector<Guardian>,
-        //expirationTime: u64, //u32
-    }
-
-    public fun init_guardian_set(admin: &signer){
-        //let x = Table::new<u8, Guardian>();
-        let x = vector::empty<Guardian>();
-        let addr = signer::address_of(admin);
-        assert!(!exists<GuardianSet>(addr), E_NO_GUARDIAN_SET);
-        move_to(admin, GuardianSet {
-             index:     0,
-             guardians: x,
-        });
-    }
+    // public fun init_guardian_set(admin: &signer){
+    //     //let x = Table::new<u8, Guardian>();
+    //     let x = vector::empty<Guardian>();
+    //     let addr = signer::address_of(admin);
+    //     //assert!(!exists<GuardianSet>(addr), E_NO_GUARDIAN_SET);
+    //     //move_to(admin, createGuardianSet(index, x));
+    // }
```

Comment:
Don't add commented out code. Delete it. If it is needed again, simply look it up in the git history.